blueprints/datamodel.tex

```diff
@@ -335,7 +335,117 @@ \subsection{Delivery Services}
 content and a set of Cache Servers and configuration options used to distribute
 that content.
 
-\subsubsection{Status}
+\subsubsection{Common Properties}
+Herein described are the properties common to all Delivery Service objects. The
+Routing Type of a Delivery Service encapsulates the methods by which clients
+may request content routing, and depending on its value the Delivery Service
+takes on a set of additional properties. Put simply, these are all different
+types of objects that are closely related.\\
+This section details all of the properties that are common to \emph{all} types of
+Delivery Services.
+
+\paragraph{Anonymous Blocking}
+A Delivery Service that has "Anonymous Blocking" tells Traffic Router to block
+requests from anonymized IP addresses. Whether or not and how well that can
+actually be done is dependent on the configuration of each Traffic Router itself,
+and if this Delivery Service is DNS-routed the only IP address Traffic Routers
+will be capable of checking for anonymization (e.g. known proxy/VPN/TOR exit node)
+then
+
+\paragraph{Deep Caching}
+
+\paragraph{Caching}
```

Comment:
Cachability might be a better word here.

Reply:
I'm a little hesitant to use made-up words. Is there any other phrase you think might be more appropriate? Like "Content Caching" or "Caching Type"?